.nf-core.yml

```diff
@@ -1,5 +1,6 @@
 bump_version: null
-lint: null
+lint:
+  actions_ci: false
 nf_core_version: 3.0.1
```

Comment:
how come you're not on 3.0.2?

Reply:
I dont know :D 3.0.2 did not pushed into this automatically 🗡️